docs/madr/decisions/049-policy-on-namespace.md

```diff
@@ -0,0 +1,529 @@
+
+As a **Frontend Service Owner** I want all requests routed to `version: v2` instances of `backend` to have 3s timeout
+so that my service meets SLO and responds withing 3s.
+
+```yaml
+apiVersion: kuma.io/v1alpha1
+kind: MeshHTTPRoute
+metadata:
+  name: backend-v2
+  namespace: backend-ns
+  labels:
+    kuma.io/zone: zone-1
+spec:
+  targetRef:
+    kind: Mesh
+  to:
+    - targetRef:
+        kind: MeshService
+        labels:
+          kuma.io/display-name: backend
+          k8s.kuma.io/namespace: backend-ns
+          kuma.io/zone: zone-1
+      rules:
+        - matches:
+            - path:
+                type: PathPrefix
+                value: /v2
+          default:
+            backendRefs:
+              - kind: MeshServiceSubset
+                name: backend
+                tags:
+                  version: v2
+---
+apiVersion: kuma.io/v1alpha1
+kind: MeshTimeout
+metadata:
+  name: backend-route-timeout
+  namespace: backend-ns
+  labels:
+    kuma.io/zone: zone-1
+spec:
+  targetRef:
+    kind: MeshHTTPRoute
+    name: backend-route # name is enough as both policies are in the same namespace
+  to:
+    - targetRef:
+        kind: Mesh
+      default:
+        http:
+          requestTimeout: 5s
+---
+apiVersion: kuma.io/v1alpha1
+kind: MeshTimeout
+metadata:
+  name: frontend-to-backend
+  namespace: frontend-ns
+  labels:
+    kuma.io/zone: zone-2
+spec:
+  targetRef:
+    kind: MeshHTTPRoute
+    labels:
+      kuma.io/display-name: backend-route
+      k8s.kuma.io/namespace: backend-ns
+      kuma.io/zone: zone-2
+  to:
+    - targetRef:
+        kind: Mesh
+      default:
+        http:
+          requestTimeout: 3s
+```
+
+### MeshGateway Examples
+
+As a **Cluster Operator** I want to deploy a single global `MeshGateway` to the cluster so that I’m able to manage a
+single deployment for the whole cluster without knowing specific details about apps and routes.
+
+```yaml
+apiVersion: kuma.io/v1alpha1
+kind: MeshGateway
+mesh: default
+metadata:
+  name: edge-gateway
+  labels:
+    kuma.io/origin: zone
+spec:
+  conf:
+    listeners:
+      - port: 80
+        protocol: HTTP
+  selectors:
+    - match:
+        kuma.io/service: edge-gateway
+```
+
+#### Targeting MeshGateway in MeshHttpRoute
+
+As a **Frontend Service Owner** I want to configure routes on `MeshGateway` without the access to
+cluster-scoped CRDs so that I could manage my routes independently.
+
+```yaml
+apiVersion: kuma.io/v1alpha1
+kind: MeshHTTPRoute
+metadata:
+  name: http-route-1
+  namespace: frontend-ns
+  labels:
+    kuma.io/origin: zone
+spec:
+  targetRef:
+    kind: MeshGateway
+    name: edge-gateway
+  to:
+    - targetRef:
+        kind: Mesh
+      rules:
+        - matches:
+            - path:
+                type: PathPrefix
+                value: /gui
+          default:
+            backendRefs:
+              - kind: MeshService
+                name: frontend
+                namespace: frontend-ns
```

Comment:
This namespace can be implicit right?
Also, if this namespace were different from `frontend-ns` would this be rejected?